backend/homework/7/src/main/java/org/example/services/VehicleService.java

```diff
@@ -0,0 +1,101 @@
+package org.example.services;
+
+import jakarta.annotation.PostConstruct;
+import org.example.entities.Speaker;
+import org.example.entities.Tyre;
+import org.example.entities.Vehicle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@Import({TyreService.class, SpeakerService.class})
+public class VehicleService {
+    private final Tyre mrfTyre;
+    private final Tyre bridgestoneTyre;
+    private final Speaker sonySpeaker;
+    private final Speaker boseSpeaker;
+    protected static final List<Vehicle> factoryOneVehicle = new ArrayList<>();
+    protected static final List<Vehicle> factoryTwoVehicle = new ArrayList<>();
+
+    @Bean(name="factoryOne")
+    List<Vehicle> vehicles1(){
+        return factoryOneVehicle;
+    }
+
+    @Bean(name="factoryTwo")
+    List<Vehicle> vehicles2(){
+        return factoryTwoVehicle;
+    }
+
+    /**
+     * Been for the most expensive vehicle in the list
+     * @return Vehicle
+     */
+    @Bean
+    Vehicle getMostExpensive(){
+        List<Vehicle> allVehicles = new ArrayList<>();
+        allVehicles.addAll(factoryOneVehicle);
+        allVehicles.addAll(factoryTwoVehicle);
```

Comment:
CRUD operations to be shifted in inventoryStore. This should be under a package `repository`